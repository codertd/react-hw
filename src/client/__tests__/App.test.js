// __tests__/App.test.js

import React from 'react';
import {
  render
} from '@testing-library/react';
import App from '../App';

it('renders', () => {
  // Mock fetch so it doesn't make a network call.
  global.fetch = jest.fn(() => Promise.resolve({ json: () => '' }));

  const { asFragment } = render(<App text="Hello!" />);
  expect(asFragment()).toMatchSnapshot();
});
