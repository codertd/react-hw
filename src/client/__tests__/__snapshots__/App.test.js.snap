// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Loading.. please wait!
    </h1>
    <img
      alt="react"
      src="[object Object]"
    />
  </div>
</DocumentFragment>
`;
